src/main/resources/assets/appliedenergistics2/lang/ru_ru.json

```diff
@@ -171,14 +176,22 @@
   "gui.appliedenergistics2.Creates": "Делает",
   "gui.appliedenergistics2.Cyan": "Бирюзовый",
   "gui.appliedenergistics2.Drive": "МЭ накопитель",
-  "gui.appliedenergistics2.ETAFormat": "HH:mm:ss",
+  "gui.appliedenergistics2.ETAFormat": "ЧЧ:мм:сс",
```

Comment:
I do not think that this will work properly, as the time format is probably expected to be in Latin letters.

Reply:
This is taken from [here](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/master/src/main/resources/assets/appliedenergistics2/lang/ru_ru.json#L180), and it works as expected there.